### PR TITLE
Docs: Updated UI of Page[Help]

### PR DESCRIPTION
**Pull request by JoyelJohny**
Before:
Screenshot 2023-10-20 at 19-08-46 Testing Library Testing Library

After:
Screenshot 2023-10-20 at 19-07-25 Testing Library Testing Library

### CODE REVIEW

**Review by alexkrolick — APPROVED** [review]

**Comment by alexkrolick** [comment]
@MatanBobi looks pretty good to me, what do you think?

File: src/pages/help.js [hunk 1]
```diff
@@ -61,7 +61,9 @@ export default function Help(props) {
         <div className="mainContainer documentContainer postContainer helpContainer">
           <div>
             <header>
-              <h1>Need help?</h1>
+              <h1>
+                <p className="headings">Need help?</p>
```
Comment (MatanBobi) [inline src/pages/help.js:65]: AFAIR, putting a `p` inside `h1` is not recommended based on the [spec](https://www.w3.org/TR/2014/REC-html5-20141028/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements). Any chance to remove this? :)

File: src/pages/help.js [hunk 2]
```diff
@@ -74,7 +76,9 @@ export default function Help(props) {
               align="left"
             />
             <section>
-              <h2>Buy a Course</h2>
+              <h1>
```
Comment (MatanBobi) [inline src/pages/help.js:79]: As an idea, having more than 1 `h1` per page isn't recommended for accessibility reasons. I think this should remain an `h2` and if we have another `h1` somewhere, we should also change it to something else.

**Review by MatanBobi — CHANGES_REQUESTED** [review]
Hi @JoyelJohny, thanks for taking the time to work on this.
As an idea, I like this change but I left a few comments regarding the implementation.

File: src/pages/help.js (hunk 1 above)
Comment (JoyelJohny) [reply src/pages/help.js:65]: Yes I can remove that, Is it okay to change `fontsize` of `h1` element.

Comment (MatanBobi) [reply src/pages/help.js:65]: Yeah, that's alright :) thanks.

**Comment by JoyelJohny** [comment]
If there is any need for changes ,Please tell me 😄

**Review by MatanBobi — APPROVED** [review]

**Comment by MatanBobi** [comment]
That's great, thanks!

**Comment by MatanBobi** [comment]
@all-contributors please add @JoyelJohny for code.

**Pull request merged**